packages/admin-panel/src/VizBuilderApp/App.jsx

```diff
@@ -44,14 +66,8 @@ export const App = ({ Footer, homeLink, logo }) => {
         <NavPanel logo={logo} homeLink={homeLink} />
 
         <Container>
-          <Switch>
-            <Route path={`${basePath}/viz-builder/:dashboardItemOrMapOverlay/new`} exact>
-              <CreateNew />
-            </Route>
-            <Route path={`${basePath}/viz-builder/:dashboardItemOrMapOverlay/:visualisationId?`}>
-              <Main />
-            </Route>
-          </Switch>
+          {/** Workaround for handling issues with this nested app */}
```

Comment:
I'm not sure if it's directly a result of upgrading, but I was having some weird, unexpected results with using `Route` in this case, I think because you have to have `Routes` as a wrapper around `Route` and that caused some headaches when trying to format the correct link because annoyingly you can't set a basepath on a `Routes` component. I suspect the solution is something with `Outlet` or something, but this was an easier solution and it works 🤷